automation/src/test/scala/org/broadinstitute/dsde/firecloud/api/Orchestration.scala

```diff
@@ -195,11 +199,23 @@ trait Orchestration extends FireCloudClient with LazyLogging {
   }
 
   object methods {
+    def createMethod(testname:String, method:Method, numSnapshots: Int = 1)
+                    (implicit token: AuthToken): String = {
+      val methodName = appendUnderscore(testname) + Util.makeUuid
+      for (i <- 1 to numSnapshots)
+        createMethod(SimpleMethod.creationAttributes + ("name"->methodName))
+      methodName
+    }
```

Comment:
I don't think this is used anywhere. Are you adding it for future use?

Reply:
it's used in "should be able to choose new method snapshot"